s6/debian-root/etc/services.d/lighttpd/finish

```diff
@@ -2,3 +2,5 @@
 
 s6-echo "Stopping lighttpd"
 killall -9 lighttpd
+service lighttpd-access-log stop
+service lighttpd-error-log stop
```

Comment:
This is closer, at least, but something odd in the output when stopping `lighttpd`:

```
pihole    | Stopping lighttpd
pihole    | Stopping lighttpd-error-log
pihole    | Stopping lighttpd-access-log
pihole    | Stopped lighttpd-error-log
pihole    | Stopped lighttpd-access-log <---------expected up to here
pihole    | Redirecting FIFO error log to stdout <------ wait, error log service started again?
pihole    | Stopping lighttpd-error-log <----- only to get stopped again
pihole    | Killing 783 <--- expected per your addition in the finish scripts for each service
pihole    | Stopped lighttpd-error-log <---- expected
```

Reply:
Ah, worked it out.. the calls to stop the two services need to happen _before_ `lighttpd` is killed so:

```
service lighttpd-access-log stop
service lighttpd-error-log stop
killall -9 lighttpd
```

Reply:
Also if we're going to have a `Stopped ` message for the other two services, perhaps we should add an `s6-echo "Stopped lighttpd" here?

Reply:
ok you have found the solution. I was wondering also with that order .... 
I also tried to replace bash command with s6 command
rm -rf => s6-rmrf
mkfifo => s6-mkfifo
chown =>s6-chown
the latter is tricky because it requires uid gid instead of a name.
not sure what is better s6 cmd or bash cmd.

Reply:
> not sure what is better s6 cmd or bash cmd.

At this stage, "whatever works" is good enough for me. I started looking at `3.0.0.0` last night, its a learning curve!

Reply:
I had a look also ;) . all services should be re desingned as the new s6 way ( s6-rc.d). At the moment, stumbled on 2 errors (cont-profile.d missing but not to be create or else $etc is wrongly set to /etc/s6-profile.d/env or something alike. The container starts but no services are running .....